src/Mono.Android/Xamarin.Android.Net/AndroidClientHandler.cs

```diff
@@ -333,6 +320,32 @@ protected void AssertSelf ()
 				Logger.Log (LogLevel.Info, LOG_APP, $"Returning");
 			return ret;
 		}
+		
+		private HttpContent GetErrorContent (HttpURLConnection httpConnection, HttpContent fallbackContent) 
```

Comment:
`private` is implied and [not necessary here](http://www.mono-project.com/community/contributing/coding-guidelines/#baroque-coding), please remove